### DISCUSSION TITLE
the weblate url of hg pull

### DISCUSSION THREAD

**Discussion by tumashu** [WeblateOrg/weblate#10721] [General]
I would like to hg pull: https://hosted.weblate.org/projects/gnu-health/, and do some manual conflict resolve, but I can not find the real hg url of weblate, any body know hg pull url of weblate?

**Comment by nijel** [comment]
It is available at https://hg.weblate.org/gnu-health/health. Weblate does not have built-in support for this, it is handled externally for the Hosted Weblate.

**Reply by tumashu** [reply]
https://hg.weblate.org/gnu-health/health has been broken, is url changed?

**Reply by nijel** [reply]
Sorry, the hgserve appears to be somewhat unstable, and we didn't notice it was not running.

I've added it to our monitoring now, but given its reliability record, we might consider discontinuing this service.

**Reply by tumashu** [reply]
thanks!

**Comment by tumashu** [comment]
Thanks very much!
